Library/Homebrew/system_config.rb

```diff
@@ -136,6 +149,7 @@ def dump_verbose_config(f = $stdout)
       f.puts "Perl: #{describe_perl}"
       f.puts "Python: #{describe_python}"
       f.puts "Ruby: #{describe_ruby}"
+      f.puts "System Ruby: #{describe_system_ruby}"
```

Comment:
In case of Git it's only using the first in PATH to a very coarse approximation. Other than that, the selection logic is a bit more involved: See top comment of #351 for details.